src/controllers/auth.js

```diff
@@ -75,4 +76,61 @@ const createDefaultTables = async () => {
   }
 };
 
-module.exports = { createDefaultTables };
+const updateSuperuser = async (fields) => {
+  const { id, password, is_superuser } = fields;
+  let newHashedPassword, newSalt;
+  let fieldsString = '';
+
+  try {
+    // find the user by using the id field
+    let user = rowService.get({
```

Comment:
Again as @IanMayo mentioned previously this variable is better to be called 'users'.